### DISCUSSION TITLE
It would be nice to implement a command to run the app in debug mode

### DISCUSSION THREAD

**Discussion by nascin** [flet-dev/flet#1398] [Ideas]
It would be nice to implement a command to run the app in debug mode

The way we could change some things, for example, use a database only for testing.

Is there any feature for this purpose?

**Comment by FeodorFitsner** [comment]
You can easily achieve that with just CLI args or environment variables. Why would you need to implement that in Flet - please elaborate?

**Reply by abcnow** [reply]
@FeodorFitsner Feodor.. can you explain how can you accomplish this with CLI arg or variables? I am currently working on an application and it is not giving me any error.. it is in blank :) I am not sure what to do here

**Reply by ndonkoHenri** [reply]
Will be nice if you could share some reproducible test code for the issue @abcnow.

**Reply by abcnow** [reply]
ok.. 
@ndonkoHenri -> here is a link : https://github.com/abcnow/test_pro/tree/main
